Jetcaster/mobile/src/main/java/com/example/jetcaster/ui/player/PlayerScreen.kt

```diff
@@ -712,6 +713,9 @@ private fun PlayerSlider(
     }
 }
 
+private fun Modifier.invisibleWhen(predicate: () -> Boolean): Modifier =
```

Comment:
Doesn't seem like this is being used in this change, but if you already have some intended uses, feel free to keep it.

Reply:
Thanks, yep I meant to remove this.